src/components/sosContacts/SosContactList.js

```diff
@@ -117,3 +107,19 @@ const styles = StyleSheet.create({
     borderRadius: 41,
   },
 });
+
+SosContactList.propTypes = {
+  contacts: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      phone: PropTypes.string,
+      message: PropTypes.string,
+    }),
+  ),
+};
+
+SosContactList.defaultProps = {
+  contacts: [
```

Comment:
why do u need default props here?

Reply:
got lint warning and it said to include defaultprops